python/lsst/cell_coadds/_GridContainer.py

```diff
@@ -59,14 +59,6 @@ def __reduce__(self) -> tuple:
 
 @continueClass
 class GridContainer:  # noqa: F811
-    def __getstate__(self) -> Dict:
-        """Support pickle with pickle.dump(s)."""
-        state: Dict[Any, Any] = {"shape": self.shape, "offset": self.offset}
-        for y in range(self.offset.y, self.offset.y + self.shape.y):
-            for x in range(self.offset.x, self.offset.x + self.shape.x):
-                state[Index2D(x, y)] = self[Index2D(x, y)]
-        return state
-
```

Comment:
This is being removed because the coverage report shows that this is not used in pickling/unpickling. This entire class is going to go away in DM-36873 (#11). I am okay with removing it here as well as removing it altogether in the next ticket.

Reply:
It'll be easier to remove in another ticket, I think, as it may snowball into removing or modifying many other things.